package.json

```diff
@@ -26,11 +26,13 @@
     },
     "homepage": "http://fluidproject.org",
     "dependencies": {
-        "eslint-plugin-jsdoc": "39.3.13"
+	"eslint": "8.53.0",
+        "eslint-plugin-jsdoc": "39.3.13",
+        "@stylistic/eslint-plugin-js": "1.0.0"
     },
     "devDependencies": {
-        "grunt": "1.5.3",
-        "grunt-eslint": "24.0.0",
+        "grunt": "1.6.1",
+        "grunt-eslint": "24.3.0",
         "grunt-jsonlint": "2.1.3"
     }
```

Comment:
This is not strictly related to this PR but I wonder if it might be worth removing `grunt` etc if it's only being used for linting this package and `eslint` is now a dependency? If we were to do so we could change the lint script to `eslint` with an appropriate glob rule to target all the JS and JSON files in this package.

Reply:
I think that's good to handle in another issue, as this small change is needed to allow any projects to work with recent versions of ESLint.

I wrote that up as #19, and left a comment there for you.  Happy to work on that once we have a chance to talk about what we want.

Reply:
I made a first pass at #19 in a separate pull (#20).